meerk40t/core/node/op_raster.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, *args, **kwargs):
             "elem rect",
             "elem line",
             "elem text",
-            "elem image",
+#            "elem image",
```

Comment:
Hm. You can raster an image though. I see the logic in excluding it but if you wanted an image and then you wanted to obscure that image with a big white heart or something to put something else in there it would need to include images.